chromium_src/BUILD.gn

```diff
@@ -616,6 +623,13 @@ source_set("browser") {
   }
 }
 
+source_set("permissions") {
```

Comment:
all source sets need the electron_config and chromium_src_config or very subtle errors can be introduced